src/main/java/org/agileware/test/PropertiesTester.java

```diff
@@ -381,7 +466,7 @@ private Method getGetter(final Class<?> type, final String propertyName, final C
      *            the property name.
      * @return the camel case concatenation of prefix and property.
      */
-    private final String getMethodName(final String prefix, final String property) {
+    private String getMethodName(final String prefix, final String property) {

```

Comment:
Any reason for removing the `final` keyword here?

Reply:
Hi, it is redundant, private methods can't be overridden so no need to add final modifier.